packages/react-native/React/Base/RCTConvert.mm

```diff
@@ -878,6 +880,38 @@ + (UIEdgeInsets)UIEdgeInsets:(id)json
   return names;
 }
 
+static RCTColorSpace defaultColorSpace = (RCTColorSpace)facebook::react::getDefaultColorSpace();
+RCTColorSpace RCTGetDefaultColorSpace(void)
+{
+  return (RCTColorSpace)facebook::react::getDefaultColorSpace();
+}
+void RCTSetDefaultColorSpace(RCTColorSpace colorSpace)
+{
+  facebook::react::setDefaultColorSpace((facebook::react::ColorSpace)colorSpace);
+}
+
++ (UIColor *)createColorFrom:(CGFloat)r green:(CGFloat)g blue:(CGFloat)b alpha:(CGFloat)a
```

Comment:
```suggestion
+ (UIColor *)RCTCreateColorWith:(CGFloat)red green:(CGFloat)green blue:(CGFloat)blue alpha:(CGFloat)alpha
```
in this way we keep the same syntax used by Apple, to be less confusing, but with the RCTPrefix we make sure to convey that it is a React Native method.